docs/decisions/0003-instructor-dashboard-rest-api-architecture.rst

```diff
@@ -0,0 +1,276 @@
+ADR-0003 Instructor Dashboard REST API Architecture
+===================================================
+
+Decision
+--------
+
+Implement unified Instructor Dashboard REST API with service orchestration
+pattern organizing 90 endpoints across 12 functional domains.
+
+API Endpoints & Services
```

Comment:
It would be useful to document what services mean in this context or link to where you've already documented it.